test/StreamableBodyTest.cpp

```diff
@@ -10,6 +10,33 @@
 using namespace ad_utility::httpUtils::httpStreams;
 using ad_utility::stream_generator::stream_generator;
 
+std::string_view toStringView(
+    const streamable_body::writer::const_buffers_type& buffer) {
+  return {static_cast<const char*>(buffer.data()), buffer.size()};
+}
+
+namespace boost {
+// For some reason those functions are required for the ASSERT_EQ/ASSERT_NE
+// macros to compile with boost::optional

```

Comment:
// When using `ASSERT_EQ/ASSERT_NE` GTest always needs `operator<<` to print the arguments in case
// of a mismatch, which are not provided by default for the following boost parameters: